<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionGraphD</class>
 <widget class="QDialog" name="FunctionGraphD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Graph</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>635</width>
     <height>485</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_funct_formula">
      <property name="text">
       <string>x ^ 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="text">
       <string>-1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_value_r_to">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>21</height>
       </size>
      </property>
      <property name="text">
       <string>Формула функции f(x):</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Draw It!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QCustomPlot" name="widget_2_graph" native="true">
      <property name="minimumSize">
       <size>
        <width>621</width>
        <height>361</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Область значения:</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>widget_2_graph</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_value_r_to</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_funct_formula</zorder>
   <zorder>pushButton</zorder>
   <zorder>lineEdit_6</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
